pkg/cli/config/initialize/basecamp.go

```diff
@@ -119,6 +119,73 @@ type BaseCampGlobals struct {
 }
 
 // Basecamp Defaults
+// master bootcmd (cloud-init user-data)
+var masterBootCMD = [][]string{
+	{"cloud-init-per", "once", "create_PART_LVM", "parted", "--wipesignatures", "-m", "--align=opt", "--ignore-busy", "/dev/md/AUX", "mkpart", "primary", "1MiB", "100%"},
+	{"cloud-init-per", "once", "create_PV", "pvcreate", "-ff", "-y", "-M", "lvm2", "/dev/md/AUX1"},
+	{"cloud-init-per", "once", "create_VG", "vgcreate", "metalvg0", "/dev/md/AUX1"},
```

Comment:
Omit the call to `parted` entirely, and change the `create_PV` and `create_VG` cloud-init commands to target `/dev/md/AUX` instead of `/dev/md/AUX1`.

Each member of `/dev/md/AUX` is already wiped and formated with a primary partition in dracut (on every NCN type): https://github.com/Cray-HPE/dracut-metal-mdsquash/blob/main/90metalmdsquash/metal-md-lib.sh#L195